src/App/App.tsx

```diff
@@ -1,32 +1,30 @@
 import React, { useState, useEffect } from 'react'
-import { Route } from 'react-router-dom'
 import './App.scss'
 import Header from '../Header/Header'
 import View from '../View/View'
-import { getCoordinates } from '../helpers/apiCalls'
 
 const App: React.FC = () => {
-	// const [ topCities, setTopCities ] = useState([])
+	const [ topCities, setTopCities ] = useState([])
 
-	useEffect(() => {getTop5Cities()}, [])
+	// useEffect(() => {getTop5Cities()}, [])
 
-	const getTop5Cities = async ():Promise<any> => {
-		const topCities = ['Denver']
-		const topCitiesData = topCities.map(city => {
-			return getCoordinates(city)
-		})
-		return topCitiesData
-	}
+	// const getTop5Cities = async ():Promise<any> => {
+	// 	const topCities = ['Denver', 'San Francisco']
```

Comment:
Good work jumping from ES5 to async await. I know that was a headache.

Reply:
Great work!